simple_test.py

```
@@ -0,0 +1,98 @@
+                if idea.reasoning:
+                    reasoning_short = (
+                        idea.reasoning[:150] + "..."
+                        if len(idea.reasoning) > 150
+                        else idea.reasoning
+                    )
+                    print(f"     💭 {reasoning_short}")
+
+        print(f"\n🎨 Total ideas generated: {len(result.synthesized_ideas)}")
+        return result
+
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)